presentation/feedprofile/src/main/java/com/hilingual/presentation/feedprofile/tab/DiaryListScreen.kt

```diff
@@ -0,0 +1,74 @@
+package com.hilingual.presentation.feedprofile.tab
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.hilingual.core.designsystem.component.content.FeedContent
+import com.hilingual.core.designsystem.theme.HilingualTheme
+import com.hilingual.presentation.feedprofile.component.FeedEmptyCard
+import com.hilingual.presentation.feedprofile.component.FeedEmptyCardType
+import com.hilingual.presentation.feedprofile.model.DiaryItem
+import com.hilingual.presentation.feedprofile.model.LikeDiaryItemModel
+import kotlinx.collections.immutable.ImmutableList
+
+@Composable
+internal fun DiaryListScreen(
+    diaries: ImmutableList<DiaryItem>,
+    emptyCardType: FeedEmptyCardType,
+    onProfileClick: (Long) -> Unit,
+    onContentClick: (Long) -> Unit,
+    onLikeClick: (Long) -> Unit,
+    onMoreClick: (Long) -> Unit,
+    onMenuClick: (Long) -> Unit,
```

Comment:
p3
같은 Long 타입의 id이더라도 userId를 넘겨야하는 경우도 있고, diaryId를 넘겨야 하는 경우도 있는데 헷갈리지 않게 명시해줘도 좋을 것 같아요.
```suggestion
    onProfileClick: (userId: Long) -> Unit,
    onContentClick: (diaryId: Long) -> Unit,
    onLikeClick: (diaryId: Long) -> Unit,
    onMoreClick: (diaryId: Long) -> Unit,
    onMenuClick: (diaryId: Long) -> Unit,
```